File: platform/sdk/python/examples/graph/graph-006_cdc_lag_monitor.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
client.cdcNew("consumer-006")
result = client.cdcLagMonitor("consumer-006")
if not result["ok"] or not isinstance(result["lagMs"], int):
    sys.exit(1)
print("XJ-OK: graph-006_cdc_lag_monitor")
sys.exit(0)

File: platform/sdk/nodejs/examples/graph/graph-006_cdc_lag_monitor.js
const { GraphClient } = require("../../mox-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
client.cdcNew("consumer-006");
const result = client.cdcLagMonitor("consumer-006");
if (!result.ok || typeof result.lagMs !== "number") process.exit(1);
console.log("XJ-OK: graph-006_cdc_lag_monitor");
process.exit(0);
